Tests/test_content.py

```diff
@@ -489,28 +488,6 @@ def run_test_scenario(t, proxy, default_test_timeout, skipped_tests_conf, nightl
              build_number, server, build_name, is_ami)
 
 
-def restart_demisto_service(ami, demisto_api_key, server):
```

Comment:
are we sure that this is not needed?

Reply:
tbh not sure... It seems to work without it. I removed it because after restarting the demisto instance, the updated content that was loaded to the server would be gone... and soooo the tests would invariably fail because various updated/new content items weren't present on the machine.

Reply:
Actually I think it makes the most sense to just make a call with the client to reset containers instead of restarting the demisto instance.